packages/integrations/preact/package.json

```diff
@@ -35,7 +35,8 @@
     "@babel/core": ">=7.0.0-0 <8.0.0",
     "@babel/plugin-transform-react-jsx": "^7.17.12",
     "babel-plugin-module-resolver": "^4.1.0",
-    "preact-render-to-string": "^5.2.0"
+    "preact-render-to-string": "^5.2.0",
```

Comment:
FYI: Recommend updating this to teh latest version (`5.2.4`), because that now calls all unmount hooks. we didn't do that before and this is necessary to cleanup signals after rendering.